modules/atomic-widgets/validators/styles-validator.php

```diff
@@ -38,49 +36,81 @@ public static function make( array $schema ): self {
 	 * }
 	 */
 	public function validate( array $styles ): array {
+		$errors_bag = [];
 		foreach ( $styles as $style_id => $style ) {
-			if ( ! isset( $style['id'] ) || ! is_string( $style['id'] ) ) {
-				$this->errors_bag[] = 'id';
+			if ( $style_id !== $style['id'] ) {
```

Comment:
What if there is not `id` in `$style` var, and what if this is not a `string` key and it is a `number`?

Reply:
then we add it to error bag,
WDYM?